### PR TITLE
Added metrics counters

### PR DESCRIPTION
**Pull request by Gobonoid**

### CODE REVIEW

File: proximo-server/metrics.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+// NewCounters returns metric counters needed for the app.
+func NewCounters() counters {
+	c := counters{
+		ErrorCounter: prometheus.NewCounter(prometheus.CounterOpts{
+			Name: "errors_total",
+			Help: "A counter of the number of errors",
+		}),
+		SinkMessagesCounter: prometheus.NewCounterVec(
```
Comment (ggirtsou) [inline proximo-server/metrics.go:21]: These counters are handled by go-pubsub automatically, please remove `SinkMessagesCounter` and `SourcedMessagesCounter`. 

See how they should be defined:
* kafka sink: https://github.com/utilitywarehouse/opsgenie-alert-producer/blob/master/main.go#L62-L69
* kafka source: https://github.com/utilitywarehouse/diff-to-kafka/blob/20b3e45608831c4b524bd6ec72150bd08ecd4128/main.go#L64-L71

**Review by ggirtsou — CHANGES_REQUESTED** [review]
Please see my comment for kafka metrics.

File: proximo-server/metrics.go (hunk 1 above)
Comment (Gobonoid) [reply proximo-server/metrics.go:21]: @ggirtsou I think that doesn't apply to this project as it's not using go-pubsub.

**Review by ggirtsou — APPROVED** [review]
Discussed with @mjgarton and we are not going to use go-pubsub for this project.

**Pull request merged**